app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -162,6 +168,15 @@ description: |
     subscription-manager attach --auto
 
   <% end -%>
+
+  <% if host_param_true?('subscription_manager_refresh', false) -%>
+  subscription-manager refresh <% if host_param_true?('subscription_manager_refresh_force', false) %> --force<% end %>
```

Comment:
Isn't `--force` completely outside the `<% %>` here? Wouldn't that make it render regardless?